src/main/java/use_case/logout/LogoutInputBoundary.java

```diff
@@ -1,4 +1,12 @@
 package use_case.logout;
 
+/**
+ * Input Boundary for the use case to switch to the login view.
+ */
```

Comment:
Simply state logout use case to be consistant with the other javadocs.

Reply:
Ok I'll simplify it

Reply:
Great!